src/index.css

```diff
@@ -52,22 +52,27 @@ code {
 
 .responsibleDecisionMakingCategory {
   background-color: var(--turtleGreen);
+  min-width: 300px;
 }
 
 .selfAwarenessCategory {
   background-color: var(--dolphinBlue);
+  min-width: 300px;
 }
 
 .socialAwarenessCategory {
   background-color: var(--giraffeYellow);
+  min-width: 300px;
 }
 
 .relationshipSkillsCategory {
   background-color: var(--tigerRed);
+  min-width: 300px;
 }
 
 .selfManagementCategory {
   background-color: var(--lightHippoPurple);
+  min-width: 300px;
```

Comment:
min-width should just be set on categoryPageButton since these other classes are used by other activity buttons and should only set the background color
<img width="738" alt="Screen Shot 2022-04-06 at 12 11 26 AM" src="https://user-images.githubusercontent.com/43453743/161894218-dd9b6082-feda-4b52-9a11-6ced3d504199.png">